tic-tac-toe/src/app/screens/Game/styles.scss

```diff
@@ -0,0 +1,8 @@
+.game {
+  display: flex;
+  flex-direction: row;
+}
+
+.gameInfo {
```

Comment:
CSS clases don't use camelCase notation. This should be called `game-info`. Same for the rest of the classes